pkg/collector/corechecks/servicediscovery/usm/service.go

```diff
@@ -151,18 +163,14 @@ func SizeVerifiedReader(file fs.File) (io.Reader, error) {
 
 // List of binaries that usually have additional process context of what's running
 var binsWithContext = map[string]detectorCreatorFn{
-	"python":    newPythonDetector,
-	"python2.7": newPythonDetector,
-	"python3":   newPythonDetector,
-	"python3.7": newPythonDetector,
-	"ruby2.3":   newSimpleDetector,
-	"ruby":      newSimpleDetector,
-	"java":      newJavaDetector,
-	"sudo":      newSimpleDetector,
-	"node":      newNodeDetector,
-	"dotnet":    newDotnetDetector,
-	"php":       newPhpDetector,
-	"gunicorn":  newGunicornDetector,
+	"python":   newPythonDetector,
+	"ruby":     newSimpleDetector,
+	"java":     newJavaDetector,
+	"sudo":     newSimpleDetector,
+	"node":     newNodeDetector,
+	"dotnet":   newDotnetDetector,
+	"php":      newPhpDetector,
+	"gunicorn": newGunicornDetector,
```

Comment:
is gunicorn a binary?

Reply:
I suggest renaming this map to something like `exeNameToCmdlineParser`